src/mondemand_statsmsg.erl

```diff
@@ -254,6 +275,57 @@ to_lwes (#md_stats_msg { send_time = SendTimeIn,
             )
   }.
 
+to_lwes (#md_stats_msg { send_time = SendTimeIn,
+                         collect_time = CollectTimeIn,
+                         prog_id = ProgId,
+                         host = Host,
+                         num_context = NumContexts,
+                         context = Context,
+                         num_metrics = NumMetrics,
+                         metrics = Metrics
+                       },
+         EventList) when NumMetrics > ?MD_MAX_METRICS ->
+  { HeadMetrics, TailMetrics } = lists:split( ?MD_MAX_METRICS, Metrics ),
+  to_lwes ( (#md_stats_msg { send_time = SendTimeIn,
+                             collect_time = CollectTimeIn,
+                             prog_id = ProgId,
+                             host = Host,
+                             num_context = NumContexts,
+                             context = Context,
+                             num_metrics = NumMetrics - ?MD_MAX_METRICS,
+                             metrics = TailMetrics
+                           }),
```

Comment:
If you bind a variable in the function head this can be shortened as well, so something like
```erlang
to_lwes (MondemandStatsMsg = #md_stats_msg { num_metrics = NumMetrics, metrics = Metrics }, EventList)
  when NumMetrics > ?MD_MAX_METRICS ->
  { HeadMetrics, TailMetrics } = lists:split( ?MD_MAX_METRICS, Metrics ),
  to_lwes ( MondemandStatsMsg#md_stats_msg {
                   num_metrics = NumMetrics - ?MD_MAX_METRICS,
                   metrics = TailMetrics },
                 [ MondemandStatsMsg#md_stats_msg {
                     num_metrics = ?MD_MAX_METRICS,
                     metrics = HeadMetrics
                   } | EventList ]);
```
(with better formatting of course :) ).